assessments/guide/security-reviewer.md

```diff
@@ -84,35 +112,55 @@ There is a possibility of a conflict of interest that can arise between a
 are important to prevent:
-
 - Individuals from intentionally or unintentionally promoting their own
 company's project
-- SIG-Security chairs and assessment leads intentionally or
+- SIG-Security chairs and review leads intentionally or
 unintentionally limiting the participation of an individual unfairly by
 asserting conflict of interest
 - Security reviews being stalled while groups belabor on who should be allowed
 to participate
 
-The conflicts of interest lie on a spectrum, and are classified into hard and soft conflicts:
+The conflicts of interest lie on a spectrum, and are classified into hard and 
+soft conflicts:
 * A hard conflict makes a reviewer ineligible to review a project.
 * A soft conflict allows a reviewer to review a project, but not as a
 [project lead](./project-lead.md).
 * It is not unusual for reviewers to have soft conflicts. The diversity of
-reviewers that are familiar with a project and can provide a deeper insight
-together with a fresh set of eyes is beneficial to the success of a security
+reviewers that are familiar with a project can provide a deeper insight
+together with a fresh set of eyes and is beneficial to the success of a security
 review.
 
-All reviewers must provide a conflict declaration to indicate which hard or soft
-conflicts do, or do not exist
-when they volunteer to be a reviewer.  This is done by placing a comment on the
-issue associated with the assessment.
-Should a conflict arise during the time of the assessment, reviewers should
-notify the lead security reviewer when
-they become aware of the potential conflict, so the new conflict may be
-consulted with the facilitator and/or chairs.
+All reviewers must provide a conflict declaration on the tracking issue to 
+indicate which hard or soft conflicts do, or do not exist when they volunteer
+to be a reviewer.  This is done by placing a comment on the issue associated 
+with the joint review using the table provided below.
+
+#### Conflict of interest statement template:
+| Hard Conflicts | Y/N |
+| :------------- | :-: |
+| Reviewer is a currently a maintainer of the project |  |
+| Reviewer is direct report of/to a current maintainer of the project |  |
+| Reviewer is paid to work on the project |  |
+| Reviewer has significant financial interest directly ties to the success of the project |  |
+
+
+| Soft Conflicts | Y/N |
+| :------------- | :-: |
+| Reviewer belongs to the same company/organization of the project, but does not work on the project |  |
+| Reviewer uses the project in his/her work |  |
+| Reviewer has contributed to the project |  |
+| Reviewer has a personal stake in the project (personal relationships, etc.) |  |
+
+### Managing conflicts
+
+Should a conflict arise during the time of the assessment, reviewers should notify
+ the lead security reviewer when they become aware of the potential conflict, 
+so the new conflict may be consulted with the facilitator and/or chairs.
+
+## Asserting conflict free readiness
```

Comment:
the team will not typically be conflict-free, so this is confusing, I propose "Asserting team readiness to conduct a balanced review" (I'll include in my PR)